Keas.Mvc/Models/ReportModels/KeyValueReportViewModel.cs

```diff
@@ -14,14 +15,17 @@ public class KeyValueReportViewModel
         public static async Task<KeyValueReportViewModel> Create(ApplicationDbContext context, string teamSlug)
         {
             var pairs = new List<KeyValues>();
-            var attributes = await context.EquipmentAttributes.Where(a => a.Equipment.Team.Slug == teamSlug).ToListAsync();
+            var attributes = await context.EquipmentAttributes.Where(a => a.Equipment.Team.Slug == teamSlug).ToListAsync();
+
+            var eav = await context.EquipmentAttributeKeys.Where(a => a.TeamId == null || a.Team.Slug == teamSlug).Select(a => a.Key).ToArrayAsync();
```

Comment:
what does eav stand for?

Reply:
@srkirkland A typo, but that was because someone didn't like my easy to remember table name :)